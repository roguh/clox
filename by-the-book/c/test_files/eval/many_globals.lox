// i = 0 ; vc("; ".join(f"var {v1}{v2} = {str(i:=i+1)}" for v1 in string.ascii_lowercase for v2 in string.ascii_lowercase if v1+v2 not in "if or"))
var aa = 1; var ab = 2; var ac = 3; var ad = 4; var ae = 5; var af = 6; var ag = 7; var ah = 8; var ai = 9; var aj = 10; var ak = 11; var al = 12; var am = 13; var an = 14; var ao = 15; var ap = 16; var aq = 17; var ar = 18; var as = 19; var at = 20; var au = 21; var av = 22; var aw = 23; var ax = 24; var ay = 25; var az = 26; var ba = 27; var bb = 28; var bc = 29; var bd = 30; var be = 31; var bf = 32; var bg = 33; var bh = 34; var bi = 35; var bj = 36; var bk = 37; var bl = 38; var bm = 39; var bn = 40; var bo = 41; var bp = 42; var bq = 43; var br = 44; var bs = 45; var bt = 46; var bu = 47; var bv = 48; var bw = 49; var bx = 50; var by = 51; var bz = 52; var ca = 53; var cb = 54; var cc = 55; var cd = 56; var ce = 57; var cf = 58; var cg = 59; var ch = 60; var ci = 61; var cj = 62; var ck = 63; var cl = 64; var cm = 65; var cn = 66; var co = 67; var cp = 68; var cq = 69; var cr = 70; var cs = 71; var ct = 72; var cu = 73; var cv = 74; var cw = 75; var cx = 76; var cy = 77; var cz = 78; var da = 79; var db = 80; var dc = 81; var dd = 82; var de = 83; var df = 84; var dg = 85; var dh = 86; var di = 87; var dj = 88; var dk = 89; var dl = 90; var dm = 91; var dn = 92; var do = 93; var dp = 94; var dq = 95; var dr = 96; var ds = 97; var dt = 98; var du = 99; var dv = 100; var dw = 101; var dx = 102; var dy = 103; var dz = 104; var ea = 105; var eb = 106; var ec = 107; var ed = 108; var ee = 109; var ef = 110; var eg = 111; var eh = 112; var ei = 113; var ej = 114; var ek = 115; var el = 116; var em = 117; var en = 118; var eo = 119; var ep = 120; var eq = 121; var er = 122; var es = 123; var et = 124; var eu = 125; var ev = 126; var ew = 127; var ex = 128; var ey = 129; var ez = 130; var fa = 131; var fb = 132; var fc = 133; var fd = 134; var fe = 135; var ff = 136; var fg = 137; var fh = 138; var fi = 139; var fj = 140; var fk = 141; var fl = 142; var fm = 143; var fn = 144; var fo = 145; var fp = 146; var fq = 147; var fr = 148; var fs = 149; var ft = 150; var fu = 151; var fv = 152; var fw = 153; var fx = 154; var fy = 155; var fz = 156; var ga = 157; var gb = 158; var gc = 159; var gd = 160; var ge = 161; var gf = 162; var gg = 163; var gh = 164; var gi = 165; var gj = 166; var gk = 167; var gl = 168; var gm = 169; var gn = 170; var go = 171; var gp = 172; var gq = 173; var gr = 174; var gs = 175; var gt = 176; var gu = 177; var gv = 178; var gw = 179; var gx = 180; var gy = 181; var gz = 182; var ha = 183; var hb = 184; var hc = 185; var hd = 186; var he = 187; var hf = 188; var hg = 189; var hh = 190; var hi = 191; var hj = 192; var hk = 193; var hl = 194; var hm = 195; var hn = 196; var ho = 197; var hp = 198; var hq = 199; var hr = 200; var hs = 201; var ht = 202; var hu = 203; var hv = 204; var hw = 205; var hx = 206; var hy = 207; var hz = 208; var ia = 209; var ib = 210; var ic = 211; var id = 212; var ie = 213; var ig = 214; var ih = 215; var ii = 216; var ij = 217; var ik = 218; var il = 219; var im = 220; var in = 221; var io = 222; var ip = 223; var iq = 224; var ir = 225; var is = 226; var it = 227; var iu = 228; var iv = 229; var iw = 230; var ix = 231; var iy = 232; var iz = 233; var ja = 234; var jb = 235; var jc = 236; var jd = 237; var je = 238; var jf = 239; var jg = 240; var jh = 241; var ji = 242; var jj = 243; var jk = 244; var jl = 245; var jm = 246; var jn = 247; var jo = 248; var jp = 249; var jq = 250; var jr = 251; var js = 252; var jt = 253; var ju = 254; var jv = 255; var jw = 256; var jx = 257; var jy = 258; var jz = 259; var ka = 260; var kb = 261; var kc = 262; var kd = 263; var ke = 264; var kf = 265; var kg = 266; var kh = 267; var ki = 268; var kj = 269; var kk = 270; var kl = 271; var km = 272; var kn = 273; var ko = 274; var kp = 275; var kq = 276; var kr = 277; var ks = 278; var kt = 279; var ku = 280; var kv = 281; var kw = 282; var kx = 283; var ky = 284; var kz = 285; var la = 286; var lb = 287; var lc = 288; var ld = 289; var le = 290; var lf = 291; var lg = 292; var lh = 293; var li = 294; var lj = 295; var lk = 296; var ll = 297; var lm = 298; var ln = 299; var lo = 300; var lp = 301; var lq = 302; var lr = 303; var ls = 304; var lt = 305; var lu = 306; var lv = 307; var lw = 308; var lx = 309; var ly = 310; var lz = 311; var ma = 312; var mb = 313; var mc = 314; var md = 315; var me = 316; var mf = 317; var mg = 318; var mh = 319; var mi = 320; var mj = 321; var mk = 322; var ml = 323; var mm = 324; var mn = 325; var mo = 326; var mp = 327; var mq = 328; var mr = 329; var ms = 330; var mt = 331; var mu = 332; var mv = 333; var mw = 334; var mx = 335; var my = 336; var mz = 337; var na = 338; var nb = 339; var nc = 340; var nd = 341; var ne = 342; var nf = 343; var ng = 344; var nh = 345; var ni = 346; var nj = 347; var nk = 348; var nl = 349; var nm = 350; var nn = 351; var no = 352; var np = 353; var nq = 354; var nr = 355; var ns = 356; var nt = 357; var nu = 358; var nv = 359; var nw = 360; var nx = 361; var ny = 362; var nz = 363; var oa = 364; var ob = 365; var oc = 366; var od = 367; var oe = 368; var of = 369; var og = 370; var oh = 371; var oi = 372; var oj = 373; var ok = 374; var ol = 375; var om = 376; var on = 377; var oo = 378; var op = 379; var oq = 380; var os = 381; var ot = 382; var ou = 383; var ov = 384; var ow = 385; var ox = 386; var oy = 387; var oz = 388; var pa = 389; var pb = 390; var pc = 391; var pd = 392; var pe = 393; var pf = 394; var pg = 395; var ph = 396; var pi = 397; var pj = 398; var pk = 399; var pl = 400; var pm = 401; var pn = 402; var po = 403; var pp = 404; var pq = 405; var pr = 406; var ps = 407; var pt = 408; var pu = 409; var pv = 410; var pw = 411; var px = 412; var py = 413; var pz = 414; var qa = 415; var qb = 416; var qc = 417; var qd = 418; var qe = 419; var qf = 420; var qg = 421; var qh = 422; var qi = 423; var qj = 424; var qk = 425; var ql = 426; var qm = 427; var qn = 428; var qo = 429; var qp = 430; var qq = 431; var qr = 432; var qs = 433; var qt = 434; var qu = 435; var qv = 436; var qw = 437; var qx = 438; var qy = 439; var qz = 440; var ra = 441; var rb = 442; var rc = 443; var rd = 444; var re = 445; var rf = 446; var rg = 447; var rh = 448; var ri = 449; var rj = 450; var rk = 451; var rl = 452; var rm = 453; var rn = 454; var ro = 455; var rp = 456; var rq = 457; var rr = 458; var rs = 459; var rt = 460; var ru = 461; var rv = 462; var rw = 463; var rx = 464; var ry = 465; var rz = 466; var sa = 467; var sb = 468; var sc = 469; var sd = 470; var se = 471; var sf = 472; var sg = 473; var sh = 474; var si = 475; var sj = 476; var sk = 477; var sl = 478; var sm = 479; var sn = 480; var so = 481; var sp = 482; var sq = 483; var sr = 484; var ss = 485; var st = 486; var su = 487; var sv = 488; var sw = 489; var sx = 490; var sy = 491; var sz = 492; var ta = 493; var tb = 494; var tc = 495; var td = 496; var te = 497; var tf = 498; var tg = 499; var th = 500; var ti = 501; var tj = 502; var tk = 503; var tl = 504; var tm = 505; var tn = 506; var to = 507; var tp = 508; var tq = 509; var tr = 510; var ts = 511; var tt = 512; var tu = 513; var tv = 514; var tw = 515; var tx = 516; var ty = 517; var tz = 518; var ua = 519; var ub = 520; var uc = 521; var ud = 522; var ue = 523; var uf = 524; var ug = 525; var uh = 526; var ui = 527; var uj = 528; var uk = 529; var ul = 530; var um = 531; var un = 532; var uo = 533; var up = 534; var uq = 535; var ur = 536; var us = 537; var ut = 538; var uu = 539; var uv = 540; var uw = 541; var ux = 542; var uy = 543; var uz = 544; var va = 545; var vb = 546; var vc = 547; var vd = 548; var ve = 549; var vf = 550; var vg = 551; var vh = 552; var vi = 553; var vj = 554; var vk = 555; var vl = 556; var vm = 557; var vn = 558; var vo = 559; var vp = 560; var vq = 561; var vr = 562; var vs = 563; var vt = 564; var vu = 565; var vv = 566; var vw = 567; var vx = 568; var vy = 569; var vz = 570; var wa = 571; var wb = 572; var wc = 573; var wd = 574; var we = 575; var wf = 576; var wg = 577; var wh = 578; var wi = 579; var wj = 580; var wk = 581; var wl = 582; var wm = 583; var wn = 584; var wo = 585; var wp = 586; var wq = 587; var wr = 588; var ws = 589; var wt = 590; var wu = 591; var wv = 592; var ww = 593; var wx = 594; var wy = 595; var wz = 596; var xa = 597; var xb = 598; var xc = 599; var xd = 600; var xe = 601; var xf = 602; var xg = 603; var xh = 604; var xi = 605; var xj = 606; var xk = 607; var xl = 608; var xm = 609; var xn = 610; var xo = 611; var xp = 612; var xq = 613; var xr = 614; var xs = 615; var xt = 616; var xu = 617; var xv = 618; var xw = 619; var xx = 620; var xy = 621; var xz = 622; var ya = 623; var yb = 624; var yc = 625; var yd = 626; var ye = 627; var yf = 628; var yg = 629; var yh = 630; var yi = 631; var yj = 632; var yk = 633; var yl = 634; var ym = 635; var yn = 636; var yo = 637; var yp = 638; var yq = 639; var yr = 640; var ys = 641; var yt = 642; var yu = 643; var yv = 644; var yw = 645; var yx = 646; var yy = 647; var yz = 648; var za = 649; var zb = 650; var zc = 651; var zd = 652; var ze = 653; var zf = 654; var zg = 655; var zh = 656; var zi = 657; var zj = 658; var zk = 659; var zl = 660; var zm = 661; var zn = 662; var zo = 663; var zp = 664; var zq = 665; var zr = 666; var zs = 667; var zt = 668; var zu = 669; var zv = 670; var zw = 671; var zx = 672; var zy = 673; var zz = 674;

print(aa);
print(mm);
print(zz);
