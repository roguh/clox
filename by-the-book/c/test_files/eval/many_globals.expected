1
324
674
